resources/profiles/Creality.ini

```diff
@@ -372,11 +394,11 @@ retract_speed = 35
 serial_port = 
 serial_speed = 250000
 single_extruder_multi_material = 0
-start_gcode = G90 ; use absolute coordinates\nM83 ; extruder relative mode\nM104 S[first_layer_temperature] ; set extruder temp\nM140 S[first_layer_bed_temperature] ; set bed temp\nM190 S[first_layer_bed_temperature] ; wait for bed temp\nM109 S[first_layer_temperature] ; wait for extruder temp\nG28 ; home all\nG92 E0.0\nG1 Z0.15 F240\nG1 X60.0 E9.0  F800.0 ; intro line\nG1 X100.0 E12.5 F800 ; intro line\nG92 E0.0
-end_gcode = M104 S0 ; turn off temperature\nM140 S0 ; turn off heatbed\nM107 ; turn off fan\n{if layer_z < max_print_height}G1 Z{z_offset+min(layer_z+30, max_print_height)}{endif} ; Move print head up\nG1 X0 F3000 ; home X axis\nM84 ; disable motors
+start_gcode = G90 ; use absolute coordinates\nM83 ; extruder relative mode\n\nM104 S[first_layer_bed_temperature] ; set extruder temp\nM140 S[first_layer_bed_temperature] ; set bed temp\n\nG28 ; home all\n;G29 ; auto bed leveling (bltouch)\n\nG1 X2 Y10 Z50 F3000 ; lift nozzle\n\nM104 S[first_layer_temperature] ; set extruder temp\nM190 S[first_layer_bed_temperature] ; wait for bed temp\nM109 S[first_layer_temperature] ; wait for extruder temp\n\nG1 X2 Y10 Z0.28 F240\nG92 E0\nG1 Y190 E15.0 F1500.0 ; intro line\nG1 X2.3 F5000\nG1 Y10 E30 F1200.0 ; intro line\nG92 E0
```

Comment:
`M104 S[first_layer_bed_temperature] ; set extruder temp` should probably use `[first_layer_temperature]` not `[first_layer_bed_temperature]`

Reply:
It absolutely should. As documented in the commit message, if we raise to first_layer_temperature, and do auto bed levelling, the nozzle may very well ooze. If we raise nozzle temp after auto bed levelling we waste time.

Raising nozzle temp to bed temp first, means the nozzle can heat up to a point where it will not ooze. Then we autohome, and optionally auto bed level (given this is a very common upgrade for ender3's), and then we move the nozzle to the front so it's easily accessible, and we raise the hotend to first_layer_temp. Then it might ooze a little, but by then the nozzle to easily accessible and makes wiping away any oozed filament easy and convenient.

This approach is pretty much near-optimal in all configurations with and without BLTouch.

Reply:
FWIW, the Prusa Mini profile hardcodes 170°C for bed leveling.

Reply:
However, for those who do not have automatic bed leveling (the majority) this causes a preheated hotend temperature to drop since marlin waits for the bed's temperature to "stabilize" even if it's already at temperature.

Reply:
Additionally, you should add all notes to the pull request description in addition to the commit messages as pull requests are usually squash-merged.

Reply:
> However, for those who do not have automatic bed leveling (the majority) this causes a preheated hotend temperature to drop since marlin waits for the bed's temperature to "stabilize" even if it's already at temperature.

Is that really the case? Since I'm only using M104 (set and continue) and not M109 (set and wait)...

Reply:
> FWIW, the Prusa Mini profile hardcodes 170°C for bed leveling.

Oh, that works too, probably slightly better with regard to speed...